ci-operator/step-registry/openshift/upgrade/qe/test/cert-rotation/openshift-upgrade-qe-test-cert-rotation-commands.sh

```diff
@@ -23,7 +32,13 @@ function extract_oc(){
 }
 
 # This step is executed after upgrade to target, oc client of target release should use as many new versions as possible, make sure new feature cert-rotation of oc amd is supported
-extract_oc
+minor_version="$(oc version --client -o json  | jq -r '.clientVersion.gitVersion' | cut -d '.' -f2)"
```

Comment:
Not use jq, there is no this command line in container. I hit the error before. Try below,
```
version=$(oc version --client | grep 'Client' | cut -d ":" -f2)
major_rel=$(echo $version | cut -d '.' -f1)
minor_rel=$(echo $version | cut -d '.' -f2)
if [[ ${major_rel} -eq 4 && ${minor_rel} -lt 10 ]];then
...
fi
```
Another thing, we also need to check major version is 4, otherwise, ocp 5.x may not  work fine.

Reply:
Sorry, I missed the below jq image included.

Reply:
I've switched to a image which contains jq, so not a problem to me.